openapi/components/requests/create_user_request.yml

```
@@ -7,8 +7,11 @@ required:
 properties:
   username:
     type: string
+    minimum: 4
+    maximum: 14
```

```suggestion
    minLength: 4
    maxLength: 14
```
`minimum` and `maximum` are for `number`.